src/services/context/context.getter.service.ts

```diff
@@ -52,4 +53,55 @@ export class ContextGetterService extends GenericGetterService {
             Constants.oneMinute(),
         );
     }
+
+    async getNftsCountForUser(address: string): Promise<number> {
+        const cacheKey = this.getCacheKey('nftsCountForUser', address);
+        const cachedData = await this.cachingService.getRemote<number>(
```

Comment:
is this cached value used somewhere else? Otherwise, you just fetch a new info from api and set it without using the cached value

Reply:
done; removed